apps/extension/server/src/diagnostics/getUnusedActionImplementations.ts

```diff
@@ -9,7 +9,7 @@ export const getUnusedActionImplementations: DiagnosticGetter = (
   machineResult,
 ) => {
   const allActions = getSetOfNames(
-    machineResult.getAllActions(['named']) || [],
+    machineResult.getAllActions(['named', 'object']) || [],
```

Comment:
The `'object'` declaration type doesn't exist:
https://github.com/statelyai/xstate-tools/blob/d9cc11dfddd888358c81b3d39372aeba482762b7/packages/machine-extractor/src/MachineExtractResult.ts#L603-L608

was this maybe intended as a draft PR? 🤔

Reply:
It does exist, that default array is just confusing https://github.com/statelyai/xstate-tools/blob/d9cc11dfddd888358c81b3d39372aeba482762b7/packages/machine-extractor/src/types.ts#L87-L92

Reply:
Used over here https://github.com/statelyai/xstate-tools/blob/d9cc11dfddd888358c81b3d39372aeba482762b7/packages/machine-extractor/src/actions.ts#L121-L142